app/src/sharedTest/java/org/oppia/app/home/HomeActivityTest.kt

```diff
@@ -81,6 +83,10 @@ import java.util.concurrent.TimeoutException
 import javax.inject.Inject
 import javax.inject.Singleton
 
+private const val EVENING_TIMESTAMP = 1556061720000
+private const val  MORNING_TIMESTAMP = 1556094120000
+private const val AFTERNOON_TIMESTAMP = 1556029320000
```

Comment:
Please include doc about  time stamp  (the time in h:mm:ss)

Reply:
Done.